environments/openshift/manifests/observatorium-template.yaml

```diff
@@ -1118,6 +1119,52 @@ objects:
         - name: telemeter-server-tls
           secret:
             secretName: telemeter-server-shared
+- apiVersion: v1
+  kind: Service
+  metadata:
+    labels:
+      app.kubernetes.io/name: memcached
+    name: memcached
+    namespace: ${NAMESPACE}
+  spec:
+    clusterIP: None
+    ports:
+    - name: memcached
+      port: 11211
+      targetPort: 11211
+    selector:
+      app.kubernetes.io/name: memcached
+- apiVersion: apps/v1beta2
+  kind: StatefulSet
+  metadata:
+    labels:
+      app.kubernetes.io/name: memcached
+    name: memcached
+    namespace: ${NAMESPACE}
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+        app.kubernetes.io/name: memcached
+    serviceName: memcached
+    template:
+      metadata:
+        labels:
+          app.kubernetes.io/name: memcached
+      spec:
+        containers:
+        - image: docker.io/memcached:1.5.20-alpine
```

Comment:
I think this needs to be templated no?

Reply:
ack, it's in the template parameters, but the parameter is not actually used in the template as a variable